native-assets/NFT/project/SPECIFICATION.md

```diff
@@ -0,0 +1,82 @@
+Table of Contents
+- [Overview](#overview)
+- [Use Cases](#use-cases)
+- [NFT Contract](#nft-contract)
```

Comment:
```suggestion
```

I meant this, similarly for the heading below

Reply:
I think "NFT contract" is a valid section. Doesn't really fall under "overview" or "use cases"